abi.html

```diff
@@ -4729,6 +4740,17 @@
 </p>
 
 <p>
+When an exception-specification (i.e., <code>noexcept</code>,
+<code>noexcept(expression)</code>, or <code>throw(type(s))</code>) is part of
+the function type, it is mangled according to
+<code>&lt;<a href="#mangle.exception-spec">exception-spec</a>&gt;</code> as
+described below.
+<code>throw()</code> should be mangled as Do, and <code>throw(anything else)</code> (as an extension) should be mangled as an empty <code>&lt;<a href="#mangle.exception-spec">exception-spec</a>&gt;</code> if it's not instantiation-dependent.
```

Comment:
I think it's unnecessary to include this line here; the amended comments in the grammar make this clear.